.github/workflows/test-suite.yml

```diff
@@ -317,17 +317,21 @@ jobs:
           ./doppelganger_protection.sh success genesis.json
   execution-engine-integration-ubuntu:
     name: execution-engine-integration-ubuntu
-    runs-on: ubuntu-latest
+    runs-on: ${{ github.repository == 'sigp/lighthouse' && fromJson('["self-hosted", "linux", "CI", "small"]') || 'ubuntu-latest'  }}
```

Comment:
I've just realized that we're one runner short of running the entire workflow in parallel after adding this one, it may be worth considering adding one more runner instance.